pkg/cmd/pr/merge/merge.go

```diff
@@ -333,10 +338,17 @@ func mergeRun(opts *MergeOptions) error {
 			if err != nil {
 				return err
 			}
+
 			err = git.CheckoutBranch(branchToSwitchTo)
 			if err != nil {
 				return err
 			}
+
+			err := PullLatestChanges(opts, baseRepo, branchToSwitchTo)
+			var execError *exec.ExitError
+			if err != nil && (errors.As(err, &execError) || (int(execError.ExitCode()) == 128)) {
```

Comment:
This looks good for catching the exit error 128 and making it non-fatal, but other types of errors should still be fatal. Currently they are silently discarded